app/src/test/java/com/example/campusguide/utils/HelperTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.campusguide.utils
+
+import org.junit.Test
+
+class HelperTest {
+
+    @Test
+    fun testCapitalizeWords() {
+
+        val lowerCaseString = "capitalize the words within this sentence"
+        val originalStrings = lowerCaseString.split(" ").toMutableList()
+        var output = ""
+        assert(originalStrings.size == 6)
+
+        for (word in originalStrings) {
+            assert(word.capitalize() != word)
+            val upperCaseWord = word.capitalize()
+            assert(upperCaseWord[0].isUpperCase())
+            output += ("$upperCaseWord ")
+
+        }
+        assert(output.substring(0, output.length - 1) == output.trim())
```

Comment:
This test doesn't appear to use our Helper class at all, it doesn't call helper.capitalizeWords() at all. 

I imagine the test being more like

`val lowerCaseString = "capitalize the words"`
`val capitalized = Helper.capitalizeWords(lowerCaseString)`
`assert(capitalized == "Capitalize The Words"`

Reply:
done!